docs/pipeline.rst

```diff
@@ -32,24 +32,131 @@ Pre-configured Pipelines
 which can be used by setting the ``pipeline`` configuration value to the name of the template you want to use.
-Here is a list of the existing templates:
-
-+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| template name        | corresponding pipeline                                                                                                                                            |
-+======================+===================================================================================================================================================================+
-| spacy_sklearn        | ``["nlp_spacy", "tokenizer_spacy", "intent_entity_featurizer_regex", "intent_featurizer_spacy", "ner_crf", "ner_synonyms",  "intent_classifier_sklearn"]``        |
-+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| mitie                | ``["nlp_mitie", "tokenizer_mitie", "ner_mitie", "ner_synonyms", "intent_entity_featurizer_regex", "intent_classifier_mitie"]``                                    |
-+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| mitie_sklearn        | ``["nlp_mitie", "tokenizer_mitie", "ner_mitie", "ner_synonyms", "intent_entity_featurizer_regex", "intent_featurizer_mitie", "intent_classifier_sklearn"]``       |
-+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| keyword              | ``["intent_classifier_keyword"]``                                                                                                                                 |
-+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| tensorflow_embedding | ``["intent_featurizer_count_vectors", "intent_classifier_tensorflow_embedding"]``                                                                                 |
-+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-
-Creating your own pipelines is possible by directly passing the names of the components to Rasa NLU in the ``pipeline``
-configuration variable, e.g.
+Here is a list of the **existing templates**:
+
+spacy_sklearn
+~~~~~~~~~~~~~
+
+to use it as a template:
+
+.. code-block:: yaml
+
+    language: "en"
```

Comment:
do we want to say smith like `your_language_here`? And also to specify that spacy supports limited number of languages

Reply:
I'll keep `en`, I think that is clear, but I've added a link to the languages section :+1: